core/metrics-core-api/src/main/java/org/hawkular/metrics/core/api/MetricsService.java

```diff
@@ -84,6 +84,15 @@
 
     Observable<Metric> findMetrics(String tenantId, MetricType type);
 
+    /**
+     *
+     * @param tenantId
+     * @param tags
+     * @param type Optional MetricType, if null is given all matching metrics are returned
```

Comment:
Shouldn't this be "if null is given, all matching user type metrics are returned"?